daphne_worker/src/durable/reports_processed.rs

```diff
@@ -45,26 +43,56 @@ pub(crate) const DURABLE_REPORTS_PROCESSED_MARK_AGGREGATED: &str =
 /// where `<report_id>` is the hex-encoded report ID.
 #[durable_object]
 pub struct ReportsProcessed {
-    #[allow(dead_code)]
     state: State,
     env: Env,
     config: DaphneWorkerConfig,
     touched: bool,
     alarmed: bool,
 }
 
```

Comment:
Please document the meaning of this constant. How did you pick 131,072? Also, is this the best name?

Reply:
that's the max amount a value can have inside a Durable Object, I got it from the error message. I'll document